docs/list.md

```diff
@@ -355,6 +355,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: [Eligible projects and issues](https://github.com/search?q=org%3Aturbot+topic%3Ahacktoberfest&type=repositories).
 - **Notes**: Read more details in the [announcement post](https://turbot.com/blog/2023/10/hacktoberfest-2023) and join our [Turbot community](https://turbot.com/community/join) `#contributors` channel with any questions.
 
+### W
+
+#### **WireMock**
+
+- **Swag**: T-Shirts, Stickers, Special swag.
+- **Requirements**:
+    - All projects in the official WireMock organization on GitHub.
+    - [WireMock Ecosystem](https://github.com/wiremock/ecosystem) projects marked with a `hacktoberfest` topic by their maintainers.
+    - Judge contributions by the most active contributors, quality and value.
+- **How to sign up**: Sign up for [registration form](https://docs.google.com/forms/d/e/1FAIpQLSfC_8mX8lpQNhksqQGUhsQ0zTvRzMC5YzufVVbVkPksxouLFw/viewform)
```

Comment:
```suggestion
- **How to sign up**: Sign up for [registration form](https://docs.google.com/forms/d/e/1FAIpQLSeBiJSYV4PNFYOH1Jo9PSheOYKxqV2NJE3LELL9a_fYq4fpfw/viewform?usp=sf_link)
```
I believe you may have the wrong Google Form link here, @biplobsd. The one that you have links to the "Treblle Hacktoberfest Hackathon." Is this a separate organization that we should also have on the Swag List?

Reply:
@crweiner I am unsure if Treblle's hacktoberfest hackathon would be counted towards the DO hacktoberfest event PR count. If thats OK, then a must add. That hackathon looks amazing, the landing page itself is so inspiring! 🎃

Reply:
> That hackathon looks amazing, the landing page itself is so inspiring! 🎃

[I just pulled it up myself and yes, that does look amazing!](https://hacked.treblle.com) 😍 

[According to their blog post](https://blog.treblle.com/treblle-hacktoberfest/), Treblle will link to your newly-created SDK, which will be in the submitter's own GitHub repo. You _can_ use your own projects to count towards Hacktoberfest, as long as the repo is opted-in and you are still making meaningful PRs.

With that in mind, I say we can add it, as long as we note that you must tag the repo for Hacktoberfest yourself. 🎃

Reply:
Forgot to ping you on the above, @vimode! ⬆️

Reply:
@crweiner 

> You can use your own projects to count towards Hacktoberfest, as long as the repo is opted-in and you are still making meaningful PRs.

Wasn't aware of this rule. Thats great. Would you be adding it in your next merge or we could create a new issue?